File: 5_odd_triangle/odd_triangle.jl
function odd_triangle(n::Int)::Nothing
    n >= 1 || error("The value must be >= 1")

    for i=1:n
        if i % 2 != 0
            println("*"^i)
        end
    end
end

print("Enter a value for the base: ")
n = readline()
n = parse(Int, n)
odd_triangle(n)

File: 5_odd_triangle/odd_triangle.cpp
#include <iostream>
#include <stdexcept>

using namespace std;

void odd_triangle(int n)
{
    if (n < 1)
    {
        throw invalid_argument("Value must be >= 1");
    };

    for (int i = 1; i <= n; i++)
    {
        if (i % 2 != 0)
        {
            for (int j = 1; j <= i; j++)
            {
                cout << "*";
            };
            cout << endl;
        }
        
    };
};

int main()
{

    cout << "Enter a number for the base: ";
    int n;
    cin >> n;
    odd_triangle(n);

    return 0;
};